packages/patternfly-3/patternfly-react/src/components/MoreInformation/MoreInformation.stories.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { withInfo } from '@storybook/addon-info';
+import { defaultTemplate } from 'storybook/decorators/storyTemplates';
+import { storybookPackageName, STORYBOOK_CATEGORY } from 'storybook/constants/siteConstants';
+import { MoreInformation } from './index';
+import { name } from '../../../package.json';
+
+const stories = storiesOf(`${storybookPackageName(name)}/${STORYBOOK_CATEGORY.WIDGETS}/MoreInformation`, module);
+stories.addDecorator(
+  defaultTemplate({
+    title: 'MoreInformation'
+  })
+);
+
+const MoreInformationContent = () => (
+  <center>Well done! The component takes 100% width by default and centers the link above.</center>
+);
```

Comment:
Better to inline the HTML in the story text below, looks confusing in the `Show Info` as it appears like `MoreInformationContent` is a supported component.